src/main/java/no/unit/alma/GetAlmaSruRecordHandler.java

```diff
@@ -44,12 +50,11 @@ public GetAlmaSruRecordHandler(AlmaSruConnection connection) {
     @Override
     @SuppressWarnings("unchecked")
     public GatewayResponse handleRequest(final Map<String, Object> input, Context context) {
-        System.out.println(input);
```

Comment:
Trengs denne for logging eller lignende?

Reply:
Det lå en lignende igjen i marc21xml-parser, tror Gregor har lagt dem inn for testing når koden ble skrevet